packages/web-workers/src/process-web-request.spec.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import 'reflect-metadata';
+
+import { Context } from '@azure/functions';
+import { ServiceConfiguration } from 'common';
+import { Logger } from 'logger';
+import { ApiController } from 'service-library';
+import { processWebRequest } from './process-web-request';
+
+let requestHandled: boolean;
+
+type TestRequestResponse = {
+    message: string;
+    controller: TestableController;
+};
+
+class TestableController extends ApiController {
+    public readonly apiVersion = '1.0';
+    public readonly apiName = 'test api name';
+    public readonly logger: Logger;
+    protected readonly serviceConfig: ServiceConfiguration;
+
+    // tslint:disable-next-line:no-any
+    protected async handleRequest(...args: any[]): Promise<TestRequestResponse> {
+        requestHandled = true;
+
+        return {
+            message: `request handled with args ${args.toString()}`,
+            controller: this,
+        };
+    }
+}
+
+describe(processWebRequest, () => {
+    let context: Context;
+
+    beforeEach(() => {
+        requestHandled = false;
+        context = ({
+            req: {
+                query: { 'api-version': '1.0' },
+            },
+        } as unknown) as Context;
+        process.env.APPINSIGHTS_INSTRUMENTATIONKEY = '00000000-0000-0000-0000-000000000000';
+        //console.log = jest.fn();
```

Comment:
Do we need this?

Reply:
Nope, I thought I took that out, my mistake